### PR TITLE
[kueuectl] Generate krew plugin configuration.

### PR DESCRIPTION
**Pull request by mbobrovskyi**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
We need to generate krew plugin configuration to update it on krew after release.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: hack/generate_krew.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    files:
+    - from: "*"
+      to: "."
+EOF
+}
+
+rm -f kueue.yaml
+
+# shellcheck disable=SC2129
+cat <<EOF >> kueue.yaml
```
Comment (alculquicondor) [inline hack/generate_krew.sh:44]:
```suggestion
cat <<EOF > kueue.yaml
```

Comment (alculquicondor) [reply hack/generate_krew.sh:44]: Also, add `kueue.yaml` to gitignore, so it's not accidentally committed. Or write to a folder that is already ignored.

**Review by alculquicondor — COMMENTED** [review]
/approve
/hold
for suggestions

File: hack/generate_krew.sh (hunk 1 above)
Comment (mbobrovskyi) [reply hack/generate_krew.sh]: Done

**Review by alculquicondor — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**